server/routes/pageFlowQueue.ts

```diff
@@ -4,6 +4,17 @@ import type { PageFlowQueue } from 'viewModels'
  * Module exposing utility functions that operate on [PageFlowQueue] instances
  */
 
+/**
+ * Adds the provided page url to the last position in the specified [PageFlowQueue].
+ * Does not change the current page (current page index).
+ */
+const addPage = (pageFlowQueue: PageFlowQueue, pageUrl: string): PageFlowQueue => {
```

Comment:
Do we need a unit test for this new method?

Reply:
Not sure that's necessary. It's also tested indirectly.

Reply:
Done, although I think we need to give careful consideration to our testing strategy: https://blog.cleancoder.com/uncle-bob/2017/10/03/TestContravariance.html